### ISSUE TITLE
PR 16379 seems to have broken the `timing-sensitive` tests in hedera services

### ISSUE THREAD

**Issue by rbarkerSL** [hashgraph/hedera-services#16492]
See the associated run here: https://github.com/hashgraph/hedera-services/actions/runs/11709318923

_Originally posted by @rbarkerSL in https://github.com/hashgraph/hedera-services/issues/16379#issuecomment-2463119631_

**Related Issue(s)**:
Relates to #16379

**Comment by imalygin** [comment]
I did a bit of an investigation.  At first I confirmed that this failure is not caused by https://github.com/hashgraph/hedera-services/pull/16379 That was expected because this PR doesn't change any logic in the code. 

To confirm that I created a `timing-sensitive-test-rerun` branch and ran the tests on it:

https://github.com/hashgraph/hedera-services/actions/runs/11731493702/job/32681700064

This run uses `hashgraph/hedera-services/.github/workflows/node-zxc-compile-application-code.yaml@refs/heads/timing-sensitive-test-rerun (3596523d2e89ba1447e87e2a859b23644eef86ae)`


Then I found the last commit that modifies `swirlds-common` module, which is much more likely to be the reason for this failure and ran the tests without this commit. The tests are passing without it:

https://github.com/hashgraph/hedera-services/actions/runs/11731776907/job/32682575988

So, most likely the issue is caused by this PR https://github.com/hashgraph/hedera-services/pull/16424
